features/importing/work_import.feature

```diff
@@ -110,6 +110,32 @@ Feature: Import Works
       And I should not see "Additional Tags:"
       And I should not see "Relationship: Detected 1/Detected 2"
 
+  Scenario: Admins see IP address on imported works
+    Given I import "http://import-site-with-tags" with a mock website
+      And I press "Post"
+    When I am logged in as an admin
+      And I go to the "Detected Title" work page
+    Then I should see "IP Address: 127.0.0.1"
+
+  Scenario: Admins see IP address on works imported without preview
+    Given I start importing "http://import-site-with-tags" with a mock website
+      And I check "Post without previewing"
+      And I press "Import"
+    When I am logged in as an admin
+      And I go to the "Detected Title" work page
+    Then I should see "IP Address: 127.0.0.1"
+
+  Scenario: Admins see IP address on multi-chapter works imported without preview
+    Given I import the urls with mock websites as chapters without preview
+      """
+      http://import-site-without-tags
+      http://second-import-site-without-tags
+      """
+    When I am logged in as an admin
+      And I go to the "Untitled Imported Work" work page
+    Then I should see "Chapters:2/2"
+    Then I should see "IP Address: 127.0.0.1"
```

Comment:
Can this second "Then" be an "And", indented two more spaces?

Reply:
:+1: I will also wait for #3153 to be merged since it hits the same import tests.